backend/dataall/core/organizations/db/organization_repositories.py

```diff
@@ -131,3 +131,7 @@ def find_group_membership(session, groups, organization):
             .first()
         )
         return membership
+
```

Comment:
very nit: instead of returning the Query object, which is a very sqlalchemy-specific class, we could return directly the list of organizations and not call `.all()` in the metadata form service